woocommerce/wc-admin
# Store profiler: Business Details steps changes

Fixes #6480

This PR adds the "MailPoet" option to the Business Details step in the OBW

### Accessibility




### Screenshots
![screenshot-one wordpress test-2021 03 03-14_45_03](https://user-images.githubusercontent.com/1314156/109850440-481c7b80-7c31-11eb-8d88-1454232a7397.png)


### Detailed test instructions:

- Create a brand new site and go to the OBW.
- In the first OBW step (`Store Details`) set a Country / Region other than `US` (e.g.: Uruguay).
- Continue with the profiler.
- In the 4th step (`Business Details`) choose any of the options in both selectors.
- Verify that the displayed extensions are:
```
Mailpoet
Facebook
Google Ads
Mailchimp
Creative Mail
```
(In that order)
- Verify that the Creative Mail option is toggled off by default.
